### ISSUE TITLE
Geo subregion logging

### ISSUE THREAD

**Issue by sagikazarmark** [sagikazarmark/curiefense-emissary-poc#13]
We need to be able to prove that access to certain subregions is blocked, so we are going to need https://github.com/curiefense/curiefense/issues/694

**Comment by mathusiast** [comment]
region and subregions are supported.
and their tags can be used for blocking

see: https://github.com/curiefense/curiefense/blob/main/curiefense/curieproxy/rust/LOGS.md

**Comment by sagikazarmark** [comment]
I assumed from the open issue that subregions are not logged, but if they are, great!

**Comment by mathusiast** [comment]
indeed, we need to update the issues. many are done. will get done right before the release.
